product_docs/docs/biganimal/release/using_cluster/02_connecting_your_cluster/03_connecting_from_pgadmin.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: Connecting from pgAdmin
+---
+
+In the pgAdmin client, you can connect to your BigAnimal cluster. The pgAdmin tool allows you to inspect, monitor, manage, and query your cluster's databases from a desktop or web UI.
+
+## Prerequisites
+
+- Sign in to [BigAnimal](portal.biganimal.com) and create a cluster. For instructions, see [Creating a cluster](../../getting_started/creating_a_cluster/). 
+
+- Install the pgAdmin tool. To install pgAdmin, see the [Download](https://www.pgadmin.org/download/) page on the pgAdmin site. 
+
+!!!Note
+   When pgAdmin opens for the first time, it asks you to set a master password. The master password secures your connections and credentials. 
+!!!
+
+## Connecting to your cluster
+
+To connect to your BigAnimal cluster, you need to enter its values into pgAdmin. Find the values in BigAnimal:
```

Comment:
I might just be a little more explicit: Find the values in BigAnimal to copy and then paste into the pgAdmin fields.